nucypher/cli/deploy.py

```diff
@@ -28,8 +28,8 @@
 from nucypher.cli import actions
 from nucypher.cli.actions import get_nucypher_password
 from nucypher.cli.actions import select_client_account
+from nucypher.cli.painting import paint_contract_deployment, paint_deployer_contract_status
```

Comment:
Sorry for nitpicking, but why not just add these imports to the list below?

Reply:
:heavy_check_mark: